src/components/erc721/collectibles/collectible_card.tsx

```diff
@@ -56,19 +51,66 @@ const Title = styled.h2`
 
 const defaultHandleClick = (e: any) => undefined;
 
-export const CollectibleCard: React.FC<Props> = (props: Props) => {
-    const { id, name, price, image, color, onClick, ...restProps } = props;
-    return (
-        <CollectibleCardWrapper
-            {...restProps}
-            id={id}
-            onClick={onClick || defaultHandleClick}
-            to={`${ERC721_APP_BASE_PATH}/collectible/${props.id}`}
-        >
-            <ImageWrapper color={color} image={image}>
-                <PriceBadge price={price} />
-            </ImageWrapper>
-            <Title>{name}</Title>
-        </CollectibleCardWrapper>
-    );
+interface OwnProps {
+    color: string;
```

Comment:
We can probably simplify the interface of this component and just pass the `Collectible`, which has the id, image, name and owner; I believe.

Reply:
Done in this [commit](https://github.com/0xProject/0x-launch-kit-frontend/pull/531/commits/da3912ff9509402867b4632ff52c2181a4015ab4)